site/content/contribute/server/developer-setup/windows-wsl.md

```diff
@@ -99,3 +116,13 @@ This is an unofficial guide. Community testing, feedback and improvements are we
     ldap_sasl_bind(SIMPLE): Can't contact LDAP server (-1)
     Makefile:102: recipe for target 'start-docker' failed
     ```
+3. If you see an error like `nc: bad address 'postgres'` when running `make run-server`and PostgreSQL's container just keeps restarting, check its logs in Docker Desktop. You should probably see something like:
+
+    ```
+    LOG: input in flex scanner failed at file "/etc/postgresql/postgresql.conf" line 1
+    FATAL: configuration file "/etc/postgresql/postgresql.conf" contains errors
+    ```
+    
+    If that's the case, `postgresql.conf` is probably a directory rather than a file, which means the volume wasn't mounted properly, so you are most likely missing some of the configuration changes in step 1.
```

Comment:
```suggestion
    If that's the case, `postgresql.conf` is probably a directory rather than a file, which means the volume wasn't mounted properly, so you are most likely missing some of the configuration changes in step 1 (linux subsystem installation).
```